pkgs/tools/X11/xplugd/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, xorg
+, pkgconfig
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xplugd";
+  version = "1.4";
+
+  src = fetchTarball {
+    url =
+      "https://github.com/troglobit/xplugd/releases/download/v1.4/xplugd-1.4.tar.xz";
+    sha256 = "1hcj5pains83gb11kr4vm58i4mfrgw7zcjh96258i3nq79xcyw93";
+  };
```

Can you use fetchFromGitHub instead? That would be preferred.
```suggestion
  src = fetchFromGitHub {
    owner = "troglobit";
    repo = "xplugd";
    rev = "v${version}";
    sha256 = "";
  };
```

---

Thank you for your suggestion, I've amended my PR to incorporate. Resolving